### DISCUSSION TITLE
Support for React 16

### DISCUSSION THREAD

**Discussion by BitlyExchange** [WalletConnect/walletconnect-monorepo#4532] [Developer Support]
Hi team. I don't know if a certain version of Web3Modal supports React 16, but I tried the Web3Modal V2 version and when I clicked the connect wallet button, it would hang on here and the button would show "connecting...". I tried to integrate "EthereumProvider" seperately and found that it was pending in the "initialize" method of "EthereumProvider".
I'm still investigating this problem. But I would like to consult about the support for react 16 at the same time.
Tks!!!

**Comment by glitch-txs** [comment]
yes we do but it's possible that you might need to update webpack & react-script

**Reply by BitlyExchange** [reply]
Appreciate your reply! Is there a minimal example about how to do this? Thanks!
